### PR TITLE
[tests-only][full-ci] Fix screenshots

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
This PR fixes the screenshot issue.
With this PR, screenshots will be reported if any webUI tests fail on retry.

## How Has This Been Tested?
- test environment: local & CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>webUITest</strong> https://drone.owncloud.com/owncloud/web/21818/8/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUITest-test-feature-Access_private_link_before_authorisation.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>webUITest</strong> https://drone.owncloud.com/owncloud/web/21821/8/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUITest-test-feature-Access_private_link_before_authorisation.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>webUITest</strong> https://drone.owncloud.com/owncloud/web/21823/8/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUITest-test-feature-044202ec-63cf-463d-b63c-364c9eff2dbb.png
  

### webUITest-test-feature-9fae2e8a-4acf-4b19-bd15-063e7750f5f1.png
  


</p></details>

File: tests/acceptance/nightwatch.conf.js [hunk 1]
```diff
@@ -33,12 +33,12 @@ const LDAP_SERVER_URL = process.env.LDAP_SERVER_URL || 'ldap://127.0.0.1'
...
-const OPENID_LOGIN = RUN_ON_OCIS || !!process.env.OPENID_LOGIN
+const OPENID_LOGIN = RUN_ON_OCIS || process.env.OPENID_LOGIN === 'true'
 const WEB_UI_CONFIG = process.env.WEB_UI_CONFIG || path.join(__dirname, 'dist/config.json')
-const SCREENSHOTS = !!process.env.SCREENSHOTS
+const SCREENSHOTS = process.env.SCREENSHOTS === 'true'
 
-const VISUAL_TEST = !!process.env.VISUAL_TEST
-const UPDATE_VRT_SCREENSHOTS = !!process.env.UPDATE_VRT_SCREENSHOTS
+const VISUAL_TEST = process.env.VISUAL_TEST === 'true'
+const UPDATE_VRT_SCREENSHOTS = process.env.UPDATE_VRT_SCREENSHOTS === 'true'
```
Comment (saw-jan) [inline tests/acceptance/nightwatch.conf.js:41]: These changes are required as env values are of type `string`.
Converting any string value (except empty string) to boolean will be `true`. So, for example, `!!'false'` is true which is not the expected one.

**Comment by saw-jan** [comment]
Screenshots are working and will be reported if any tests fail on retry
see: https://github.com/owncloud/web/pull/6244#issuecomment-1011898467

**Review by SwikritiT — COMMENTED** [review]
LGTM

**Comment by phil-davis** [comment]
fix-screenshots

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingAccept</strong> https://drone.owncloud.com/owncloud/web/21859/15/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingAcceptShares-acceptShares-feature-6a8e9f6b-f8fb-4041-ade9-8a9feedd2e2b.png
  

### webUISharingAcceptSharesToRoot-acceptShares-feature-9121b110-8a2b-4121-975c-2a3eb35656a9.png
  


</p></details>

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/web/21859/15/21
`webUISharingAcceptSharesToRoot/acceptShares.feature:108` failed
Issue #6097 

Restarting drone CI

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternal3</strong> https://drone.owncloud.com/owncloud/web/21875/59/1
The following scenarios passed on retry:
- webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:136

**Comment by kiranparajuli589** [comment]
> Results for **oCISSharingInternal3** https://drone.owncloud.com/owncloud/web/21875/59/1 The following scenarios passed on retry:
> 
> * webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:136

is this reported?

**Comment by saw-jan** [comment]
@phil-davis @kiran I need to look at it more to better report which tests the screenshot is for.
I found out that whenever a test fails in a pipeline, screenshots of all the tests that are expected to fail will also get reported.
We may not want those screenshots.

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp1</strong> https://drone.owncloud.com/owncloud/web/21923/68/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIPrivateLinks-accessingPrivateLinks_feature-L17.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortrait1</strong> https://drone.owncloud.com/owncloud/web/21933/43/1
The following scenarios passed on retry:
- webUIPrivateLinks/accessingPrivateLinks.feature:9

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp1</strong> https://drone.owncloud.com/owncloud/web/21933/68/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIPrivateLinks-accessingPrivateLinks_feature-L17.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingIntGroupsToRoot</strong> https://drone.owncloud.com/owncloud/web/21948/29/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalGroupsToRootEdgeCases-shareWithGroupsEdgeCases_feature-L37.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingIntGroups</strong> https://drone.owncloud.com/owncloud/web/21948/28/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingInternalGroupsEdgeCases-shareWithGroupsEdgeCases_feature-L42.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10Basic2</strong> https://drone.owncloud.com/owncloud/web/21948/13/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIWebdavLockProtection-move_feature-L100.png
  

### webUIWebdavLockProtection-move_feature-L101.png
  

### webUIWebdavLockProtection-upload_feature-L31.png
  

### webUIWebdavLockProtection-upload_feature-L32.png
  

### webUIWebdavLockProtection-upload_feature-L51.png
  

### webUIWebdavLockProtection-upload_feature-L52.png
  

### webUIWebdavLockProtection-upload_feature-L71.png
  

### webUIWebdavLockProtection-upload_feature-L72.png
  

### webUIWebdavLockProtection-upload_feature-L90.png
  

### webUIWebdavLockProtection-upload_feature-L91.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingFolderPermissions</strong> https://drone.owncloud.com/owncloud/web/21948/27/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingFolderPermissionsGroups-sharePermissionsGroup_feature-L61.png
  

### webUISharingFolderPermissionsGroups-sharePermissionsGroup_feature-L62.png
  

### webUISharingFolderPermissionsGroups-sharePermissionsGroup_feature-L63.png
  

### webUISharingFolderPermissionsGroups-sharePermissionsGroup_feature-L64.png
  

### webUISharingFolderPermissionsGroups-sharePermissionsGroup_feature-L65.png
  

### webUISharingFolderPermissionsGroups-sharePermissionsGroup_feature-L66.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingFilePerm</strong> https://drone.owncloud.com/owncloud/web/21948/25/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingFilePermissionMultipleUsers-shareFileWithMultipleUsers_feature-L62.png
  

### webUISharingFilePermissionMultipleUsers-shareFileWithMultipleUsers_feature-L63.png
  

### webUISharingFilePermissionMultipleUsers-shareFileWithMultipleUsers_feature-L64.png
  

### webUISharingFilePermissionMultipleUsers-shareFileWithMultipleUsers_feature-L65.png
  

### webUISharingFilePermissionMultipleUsers-shareFileWithMultipleUsers_feature-L66.png
  

### webUISharingFilePermissionMultipleUsers-shareFileWithMultipleUsers_feature-L67.png
  

### webUISharingFilePermissionsGroups-sharePermissionsGroup_feature-L62.png
  

### webUISharingFilePermissionsGroups-sharePermissionsGroup_feature-L63.png
  

### webUISharingFilePermissionsGroups-sharePermissionsGroup_feature-L64.png
  

### webUISharingFilePermissionsGroups-sharePermissionsGroup_feature-L65.png
  

### webUISharingFilePermissionsGroups-sharePermissionsGroup_feature-L66.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingPermToRoot</strong> https://drone.owncloud.com/owncloud/web/21948/36/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingPermissionToRoot-shareAdvancePermissionsGroup_feature-L71.png
  

### webUISharingPermissionToRoot-shareFileWithMultipleUsers_feature-L66.png
  

### webUISharingPermissionToRoot-shareFolderWithMultipleUsers_feature-L67.png
  

### webUISharingPermissionToRoot-sharePermissionsFileGroup_feature-L65.png
  

### webUISharingPermissionToRoot-sharePermissionsFoldersGroup_feature-L67.png
  

### webUISharingPermissionToRoot-sharedFolderWithMultipleUsersAdvancedPermissions_feature-L70.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10Basic1</strong> https://drone.owncloud.com/owncloud/web/21948/12/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIWebdavLocks-locks_feature-L274.png
  

### webUIWebdavLocks-locks_feature-L275.png
  

### webUIWebdavLocks-locks_feature-L298.png
  

### webUIWebdavLocks-locks_feature-L299.png
  

### webUIWebdavLocks-locks_feature-L323.png
  

### webUIWebdavLocks-locks_feature-L324.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternalRoot</strong> https://drone.owncloud.com/owncloud/web/21948/41/1
The following scenarios passed on retry:
- webUISharingExternalToRoot/federationSharing.feature:111
- webUISharingExternalToRoot/federationSharing.feature:378

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingFolderAdvPermsGrp</strong> https://drone.owncloud.com/owncloud/web/21948/26/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L63.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L64.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L65.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L66.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L67.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L68.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L69.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L70.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L71.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L72.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L73.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L74.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp1</strong> https://drone.owncloud.com/owncloud/web/21948/68/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIPrivateLinks-accessingPrivateLinks_feature-L17.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingAccept</strong> https://drone.owncloud.com/owncloud/web/21951/15/1
The following scenarios passed on retry:
- webUISharingAcceptSharesToRoot/acceptShares.feature:108

File: .drone.star [hunk 2]
```diff
@@ -151,7 +151,7 @@ config = {
                 "WEB_UI_CONFIG": "%s/dist/config.json" % dir["web"],
             },
             "visualTesting": False,
-            "screenShots": False,
+            "screenShots": True,
```
[the remark this answers is not in the document]
Comment (pascalwengerter) [reply .drone.star:155]: @kulmann what do you think?

[the remark this answers is not in the document]
Comment (kulmann) [reply .drone.star:155]: Screenshots on error by default is fine by me.

**Review by phil-davis — COMMENTED** [review]
LGTM. I will give some time for others to review and comment, then give a real green tick and we can merge.

**Review by kulmann — APPROVED** [review]

**Review by pascalwengerter — APPROVED** [review]
No strong preference from my side. I doubt that screenshots will provide a big improvement in debugging acceptance tests, but willing to give it a try therefore LGTM!

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
That was quick feedback. I will merge, and we will see the result in the next few days as we get fails.

**Pull request merged**